whatslab/src/App.css

```diff
@@ -0,0 +1,38 @@
+.App {
```

Comment:
Acho que esse arquivo não tá sendo mais usado em lugar nenhum né? Se for o caso, pode apagar!